### ISSUE TITLE
podcast support

### ISSUE THREAD

**Issue by shlyakpavel** [xou816/spot#384]
Hello,
Thanks for a great project! This is the only stable and reliable way to listen to Spotify on aarch64 devices like Pinebook Pro. I enjoy it really really much.
However, I couldn't find a way to listen to spotify podcasts yet. I'd really love seeing podcast support in this project
Sincerely,
Pavel

**Comment by jannuary** [comment]
Duplicate of #95

Either way, I don't think it's a good fit for Spot, a music app, to also play podcasts - while, yes, Spotify does it, it provides a sub-par experience and imho, it makes more sense to provide a more focused UX.

**Comment by CleoMenezesJr** [comment]
Well, the app description is "Native Spotify client for the GNOME". People wait to Spot what Spotify offers, and it's not just music.

**Comment by jannuary** [comment]
> Well, the app description is "Native Spotify client for the GNOME Desktop". People expect on Spot what Spotify offers, and it's not just music.

[Is it?](https://flathub.org/apps/details/dev.alextren.Spot)

**Comment by CleoMenezesJr** [comment]
> [Is it?](https://flathub.org/apps/details/dev.alextren.Spot)

Yep!
![image](https://user-images.githubusercontent.com/54215258/201538001-259d4c5d-04a6-4eed-8d63-dbb43bdd40a6.png)

**Comment by jannuary** [comment]
Well, in that case it should be changed on the repo side as well.

**Comment by davidsmorais** [comment]
Is there any technical limitation preventing podcasts from being shown ?

**Comment by jannuary** [comment]
It hasn't changed since last time – it's blocked by design, unless the new batch of contributors have different thoughts.
